compare_usafacts_nyt_083120.Rmd

```diff
@@ -0,0 +1,427 @@
+
+# load data
+dat_nyt <- read_csv(file.path("data","NYT_data_090120.csv"))
+
+dat_usafacts_cases <- read_csv(file.path("data","covid_confirmed_usafacts_090120.csv")) %>%
+  gather(date, cases, names(.)[grepl("^\\d",names(.))]) %>%
+  rename(fips=countyFIPS, county='County Name', state=State) %>%
+  #filter(!fips %in% 0:1) %>%
+  mutate(fips = str_pad(fips, width=5, pad="0"),
+         date = mdy(date))
+
+dat_usafacts_deaths <- read_csv(file.path("data","covid_deaths_usafacts_090120.csv")) %>%
+  gather(date, deaths, names(.)[grepl("^\\d",names(.))]) %>%
+  rename(fips=countyFIPS, county='County Name', state=State) %>%
+  #filter(!fips %in% 0:1) %>%
+  mutate(fips = str_pad(fips, width=5, pad="0"),
+         date = mdy(date))
+```
+
+# {.tabset}
+There are several national-level aggregations of county-level COVID case and death data that researchers have to choose from. Each source has its own data collection methods, which can result in variations in numbers across sources.
+
+On this page, we explore to what extent the following two data sources for county-level cumulative counts of confirmed COVID-19 cases and deaths are comparable. In our comparison, we focus on cumulative case counts as of 8/31/2020 (data accessed on 9/01/2020).
+
+**New York Times (NYT)**: The New York times keeps maintains a [data set](https://github.com/nytimes/covid-19-data) reporting daily cumulative counts of COVID-19 cases and deaths.
+
+**USAFacts**: USAFacts also maintains a [data set](https://usafacts.org/visualizations/coronavirus-covid-19-spread-map/) reporting daily cumulative counts of COVID-19 cases and deaths.
+
+The data for each of the two sources are gathered from state and local public health agencies, however there are some differences between the two sources in how the data is reported. This [blog post](https://source.opennews.org/articles/comparison-four-major-covid-19-data-sources/) reviews some of the major differences, for example:
+
+1) **Time of update:** NYT is updated once daily around 8am EDT with case and death counts from the previous day. USAFacts is updated once daily around 12am PDT (3am EDT) for cases from the previous day. Because USAFacts is updated before NYT, USAFacts generally reports lower case and death counts than NYT for a given day.
+
+2) **Geographic exceptions:** NYT reports cases for all five boroughs of New York city as a single area ("New York City"). USAFacts reports New York City cases separately for all five counties, though also has a category for “New York City Unallocated/Probable”. NYT reports all case for Kansas City in a single group ("Kansas City"). The case counts for the four counties that include parts of Kansas City represent only the parts of the counties that are outside of Kansas City. In the USAFacts data, all Kansas City cases are allocated to Jackson County.
+
+3) **Other differences:** See the blog post linked above for more information on the differences between the data sources.
+
```

Comment:
in all these figures with bigger numbers can we use scale_y_continuous(label=comma)?